src/pomelo_fmt_plug.c

```diff
@@ -3,6 +3,9 @@
  * contest by Dhiru Kholia.
  */
 
```

Comment:
`#include "arch.h"`

Reply:
Maybe this is not needed. 

In the output of `make` I see `gcc -DAC_BUILT ... -DARCH_LITTLE_ENDIAN=1 ...`. It seems that the code will "know" `ARCH_LITTLE_ENDIAN` value even without including `arch.h`.

Reply:
Oh, that's jumbo-specific. I didn't recall. Anyway, I suggest that you do include `arch.h`, such as for `Makefile.legacy` to work.